deploy_website.sh

```diff
@@ -40,9 +40,13 @@ cp CONTRIBUTING.md docs/contributing.md
 # Build the site and push the new files up to GitHub
 mkdocs gh-deploy
 
+# Undo any file changes to be able to jump branches.
+git checkout -- .
```

Comment:
would `git reset --soft` suffice?

Reply:
No, gotta be `--hard`. What's the difference anyway?

Reply:
`reset --soft` returns to the desired hash while retaining the changes from all the commits between the `HEAD` and desired hash.

Reply:
I'm trying to undo the changes introduced by the previous commands in this script, and `git reset --soft` doesn't seem to cut it. `git checkout -- .` works, so I'll keep it.

Reply:
Yes, you need a hard reset here, or a stash!